addons/enemybar/enemybar.lua

```diff
@@ -24,31 +24,524 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 --]]
 
 _addon.name = 'enemybar'
-_addon.author = 'mmckee'
-_addon.version = '1.0.2'
+_addon.author = 'mmckee,akaden'
+_addon.version = '1.1.0'
 _addon.language = 'English'
+_addon.commands = {'enemybar','eb'}
 
 config = require('config')
 images = require('images')
 texts = require('texts')
-require('gui_settings')
-require('targetBar')
-require('subtargetBar')
+table = require('table')
+require 'bars'
+require 'actionTracking'
```

Comment:
If you care for my feelings at all, please surround the required module names with parentheses :(